lib/Doctrine/ORM/EntityRepository.php

```diff
@@ -185,7 +185,7 @@ public function findAll()
      * @param int|null $offset
      *
      * @psalm-param array<string, mixed> $criteria
-     * @psalm-param list<string>|null $orderBy
+     * @psalm-param list<string>|array<string, string>|null $orderBy
```

Comment:
Good catch, my bad! Should `list<string>` be removed too? Also, `EntityPersister::loadAll()` seems to have the same issue…

Reply:
If `list<string>` can not be, so it can be removed

Reply:
Should i update `EntityPersister::loadAll()`?

Reply:
Yes please